eessi_bot_event_handler.py

```diff
@@ -23,10 +24,25 @@
 
 from pyghee.lib import PyGHee, create_app, get_event_info, read_event_from_json
 from pyghee.utils import log
-import os
 
 
 class EESSIBotSoftwareLayer(PyGHee):
+    def __init__(self):
+        super().__init__()
+        cfg = config.read_config()
+        event_handler_cfg = cfg['event_handler']
+        self.logfile = event_handler_cfg.get('log_path')
+
+    def log(self, msg):
```

Comment:
Would be nicer to support "lazy logging", where formatting is only done when logging is done:

```python
def log(self, msg, *args):
    """..."""
    if args:
        msg = msg % args
```

then you can use:

```python
self.log("example: %s (%s)", foo, bar)
```

instead of having to pre-format the message:

```python
self.log("example: %s (%s)" % (foo, bar))